.github/workflows/release-beta.yaml

```
@@ -0,0 +1,37 @@
+# .github/release.yml
+
+name: Release betas
+on:
+  push:
+    branches:
+      - v4 # release on all commits to v4
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)